estimatron.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import os
+import logging
+import click # TODO /usr/local/bin/python3 -m pip install click==8.0.1
+import coloredlogs # TODO /usr/local/bin/python3 -m pip install coloredlogs==15.0.1
+from prettytable import PrettyTable # TODO /usr/local/bin/python3 -m pip install prettytable==2.2.0
+
+logging_level = os.environ.get('LOGGING_LEVEL', logging.INFO)
+
+coloredlogs.install(fmt='%(asctime)s %(levelname)s %(message)s', datefmt="%m/%d/%Y %I:%M:%S %p %Z")
+coloredlogs.set_level(logging_level)
+logging.basicConfig(level=coloredlogs.get_level())
+logging.getLogger().setLevel(logging_level)
+
+complexity_level = 0
+table = PrettyTable()
+title = "Software projects sorted by increasing difficulty:"
+goals = [
```

Comment:
Add a list of options that will allow decreasing the score. Including but not limited to:

* Documentation.
* Tests coverage.